src/pcap/config/config.example.yml

```diff
@@ -44,13 +44,24 @@ api:
       private_key: cc-client-key.key
       ca: cc-ca.pem
   bosh: # -> *pcap.BoshHandler `yaml:omitempty`
```

Comment:
consider extending `Environment` with the `AgentPort` or adding a property here. Probably it's easier and more flexible to add the AgentPort to `Environment`. We can also make a new composite struct that contains an `Environment` and additional things.

We can then write out the pure `Environment` bit of it when writing out the `~/.bosh/config` file in the pcap-bosh-cli.